src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -117,11 +117,22 @@ void Instance::InvokeCommand(HandlerContext & ctxt)
             ctxt, [this](HandlerContext & ctx, const auto & req) { HandleRemoveNetwork(ctx, req); });
         return;
 
-    case Commands::ConnectNetwork::Id:
+    case Commands::ConnectNetwork::Id: {
         VerifyOrReturn(mFeatureFlags.Has(Feature::kWiFiNetworkInterface) || mFeatureFlags.Has(Feature::kThreadNetworkInterface));
+#if CONFIG_NETWORK_LAYER_BLE
+        // If commissionee does not support Concurrent Connections, request the BLE to be stopped.
+        // Start the ConnectNetwork, but this will not complete until the BLE is off.
+        bool supportsConcurrentConnection = CHIP_DEVICE_CONFIG_SUPPORTS_CONCURRENT_CONNECTION;
+        if (!supportsConcurrentConnection)
+        {
+            ChipLogProgress(NetworkProvisioning, "Commands::ConnectNetwork::Id PostCloseAllBLEConnectionsToOperationalNetworkEvent");    
```

Comment:
Consider using more user-friendly and concise logs to make them intelligible even for non-developers. Sth like: "Closing BLE connections due to non-concurrent mode"

Reply:
Logs updated.